http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -881,61 +881,6 @@ paths:
             - email
             - profile
   "/api/cluster/{googleProject}/{clusterName}":
-    patch:
-      deprecated: true
-      summary: Updates the configuration of a cluster.
-      description: In order to update the configuration of a cluster, it must first be running or paused
-      operationId: updateCluster
-      tags:
-        - cluster
-      parameters:
-        - in: path
-          name: googleProject
-          description: googleProject
-          required: true
-          schema:
-            type: string
-        - in: path
-          name: clusterName
-          description: clusterName
-          required: true
-          schema:
-            type: string
-      requestBody:
-        content:
-          application/json:
-            schema:
-              $ref: "#/components/schemas/UpdateClusterRequest"
```

Comment:
Can `"#/components/schemas/UpdateClusterRequest"` be deleted too?

Reply:
oh yea..will do